data/json/itemgroups/magazines.json

```diff
@@ -48,10 +48,10 @@
       ["m1911mag", 10],
       ["m1918mag", 5],
       ["m9mag", 10],
-      ["stenmag", 40],
       ["ppsh35mag", 15]
```

Comment:
These are still not military surplus magazines in the context of the game setting.